pallets/liquidity-rewards/src/lib.rs

```diff
@@ -55,92 +56,39 @@ pub use frame_support::{
 use frame_system::pallet_prelude::*;
 use num_traits::sign::Unsigned;
 pub use pallet::*;
-use sp_runtime::{
-	traits::{BlockNumberProvider, Saturating, Zero},
-	FixedPointOperand,
-};
+use sp_runtime::{traits::Zero, FixedPointOperand};
 use sp_std::mem;
 use weights::WeightInfo;
 
-/// Type that contains the timestamp of an epoch.
-#[derive(Encode, Decode, TypeInfo, MaxEncodedLen, RuntimeDebug)]
-#[cfg_attr(test, derive(PartialEq))]
-pub struct EpochTimestamp<BlockNumber>(BlockNumber);
```

Comment:
We no longer need a custom type because now the default value doesn't need to do any computation.